src/wasm-config.h.in

```diff
@@ -91,6 +91,14 @@
 #define PRIzd "zd"
 #define PRIzx "zx"
 
+/* WABT_BUILTIN_UNREACHABLE - Expands to an expression which states
+ * that it is undefined behavior for the compiler to reach this
+ * point.  Otherwise is not defined.
+ */
+#if __has_builtin(__builtin_unreachable)
+# define WABT_BUILTIN_UNREACHABLE __builtin_unreachable()
```

Comment:
What about just WABT_UNREACHABLE?  (since the fact that its got builtin in the name is a gcc implementation detail here)